lib/management/poll.js

```diff
@@ -97,6 +97,9 @@ function getStats() {
 }
 
 function pushStats(managementEndpoint, instanceId, remoteToken, next) {
+    if (process.env.PUSH_STATS && process.env.PUSH_STATS === 'false') {
```

Comment:
The `process.env.PUSH_STATS && ` test is redundant, `undefined` just won't verify the `=== 'false'` condition